schema-loader/src/integration-test/java/com/scalar/db/schemaloader/cosmos/CosmosSchemaLoaderIntegrationTest.java

```diff
@@ -0,0 +1,71 @@
+package com.scalar.db.schemaloader.cosmos;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.schemaloader.SchemaLoaderIntegrationTestBase;
+import com.scalar.db.schemaloader.core.SchemaOperator;
+import com.scalar.db.schemaloader.schema.SchemaParser;
+import com.scalar.db.storage.cosmos.CosmosConfig;
+import com.scalar.db.storage.cosmos.CosmosEnv;
+import java.io.FileOutputStream;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Properties;
+
+public class CosmosSchemaLoaderIntegrationTest extends SchemaLoaderIntegrationTestBase {
+  private static final CosmosConfig config = CosmosEnv.getCosmosConfig();
+
+  @Override
+  protected void initialize() throws Exception {
+    Properties properties = config.getProperties();
+    try (final FileOutputStream fileOutputStream = new FileOutputStream(CONFIG_FILE)) {
+      properties.store(fileOutputStream, null);
+    }
+  }
+
+  @Override
+  protected void parsingTables() throws Exception {
+    tables =
+        SchemaParser.parse(
+            Paths.get(SCHEMA_FILE),
+            ImmutableMap.of(SchemaOperator.NAMESPACE_PREFIX, CosmosEnv.getDatabasePrefix().get()));
```

Comment:
Please care about the case where the database prefix doesn't exist.

Reply:
Updated in [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/a092f40750397b43dba8e63a10aa802d77b36050)

Reply:
Updated in [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/ffc130bde7d8d47ce6d2e9273fe6eef7c5f4bd2d)